src/main/java/com/regnosys/rosetta/common/serialisation/RosettaObjectMapper.java

```diff
@@ -12,17 +18,24 @@ public class RosettaObjectMapper {
     private RosettaObjectMapper() {
     }
 
-    public static ObjectMapper getNewMinimalRosettaObjectMapper() {
-        return new AnnotationBasedObjectMapperCreator(false).create()
-                //This call is scans the classpath and can be slow. Use the AnnotationBasedObjectMapperCreator directly if you don't need it.
-                .findAndRegisterModules();
+    public static ObjectMapper getRosettaJSONMapper() {
+        return new RosettaJSONObjectMapperCreator(false).create();
     }
 
-    /**
-     * Creating new RosettaObjectMapper instances is expensive, use the singleton instance if possible.
-     */
-    public static ObjectMapper getNewRosettaObjectMapper() {
-        return getNewMinimalRosettaObjectMapper()
+    public static ObjectMapper getOptimizedRosettaJSONMapper() {
```

Comment:
This will impact every place that any community user of`getNewRosettaObjectMapper`. If we want to remove it, we need to deprecate and introduce the new one.